main.go

```diff
@@ -1,18 +1,52 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"log"
+	"os"
 )
 
 const gomodName = "go.mod"
 
 func main() {
+	if err := run(); err != nil {
+		log.Fatal(err)
+	}
+}
 
+func run() error {
 	updatedContents, err := MergeRequires(gomodName)
 	if err != nil {
-		log.Fatal(err)
+		return fmt.Errorf("failed to merge requires: %w", err)
+	}
+
+	var inplace bool
+	flag.BoolVar(&inplace, "in-place", false, "replace the contents of go.mod with the updated contents")
+	flag.BoolVar(&inplace, "i", false, "replace the contents of go.mod with the updated contents")
+	flag.Parse()
```

Comment:
One of the problems with the default flags package is that you get the same line output twice with an invalid flag. Cobra can fix this - or we could update `-i` to read `see `--in-place`

Reply:
I think it's fine for now.